assets/js/blocks-registry/payment-methods/payment-method-config.ts

```diff
@@ -46,15 +70,19 @@ export default class PaymentMethodConfig {
 	}
 
 	// canMakePayment is calculated each time based on data that modifies outside of the class (eg: cart data).
-	get canMakePayment() {
+	get canMakePayment(): (
+		canPayArgument: CanMakePaymentArgument
+	) => boolean {
```

Comment:
We have a type that describes this function :)
```suggestion
	get canMakePayment(): CanMakePayment {
```